services/release/release.go

```diff
@@ -49,6 +49,15 @@ func createTag(gitRepo *git.Repository, rel *models.Release) error {
 		if err != nil {
 			return fmt.Errorf("CommitsCount: %v", err)
 		}
+		// Prepare Notify
+		if err := rel.LoadAttributes(); err != nil {
```

Comment:
It seems you should move these notify to line 39 followed `createTag` ?

Reply:
You are right, how dumb I am... :sweat_smile: